process.c

```diff
@@ -3035,8 +3038,23 @@ NORETURN(static VALUE f_exec(int c, const VALUE *a, VALUE _));
  *
  *  \String argument +command_line+ is a command line to be passed to a shell;
  *  it must begin with a shell reserved word, begin with a special built-in,
- *  or contain meta characters.
- *  It may also contain arguments and options for that command.
+ *  or contain meta characters:
+ *
+ *    exec('echo')                         # Built-in.
+ *    exec('if true; then echo "Foo"; fi') # Shell reserved word.
+ *    exec('date > date.tmp')              # Contains meta character.
+ *
+ *  Output:
+ *
+ *    Sat Aug 26 09:26:05 AM CDT 2023
```

Comment:
This output is for the 3rd one only?
Since `date` is redirected, no output.

Reply:
Duh.  Removed.